addon/adapters/pouch.js

```diff
@@ -184,8 +209,7 @@ exports["default"] = DS.RESTAdapter.extend({
   deleteRecord: function (store, type, record) {
     this._init(type);
     var data = this._recordToData(store, type, record);
-    return this.db.rel.del(type.typeKey, data).then(function () {
-      return ''; // ember doesn't like getting a json response of {deleted: true}
-    });
+    return this.db.rel.del(type.typeKey, data)
+      .then(extractDeleteRecord);
```

Comment:
suppose the Ember bug noted in the comment has been fixed by now? Else I don't see any harm in keeping it in.